packages/client/src/__tests__/integration/happy/not-so-exhaustive-schema-mongo/common.ts

```diff
@@ -9,15 +9,14 @@ if (isMacOrWindowsCI) {
   jest.setTimeout(80_000)
 }
 
-export const testGeneratedClient = (runtimeName: 'binary' | 'library' | 'dataProxy') => async () => {
+export const testGeneratedClient = (runtimeName: 'binary' | 'library') => async () => {
   const clientDir = path.join(__dirname, 'test-clients', runtimeName)
   await fs.promises.mkdir(clientDir, { recursive: true })
   await fs.promises.copyFile(path.join(__dirname, 'schema.prisma'), path.join(clientDir, 'schema.prisma'))
 
   const generatorOpts = {
```

Comment:
I think we can now pass `engineType: ClientEngineType` as the argument of this function instead of `runtimeName` and get rid of this block

Reply:
Done.

Reply:
Also applied everywhere else where I could.

Reply:
Then we can now remove `generatorOpts` and replace `...generatorOpts` below with `{ engineType: runtimeName }`.

Reply:
That was my thinking initially but I didn't word the comment very clearly, and github doesn't allow adding a code suggestion here 😄

Reply:
Oh right, thanks for pointing this out. Now fixed!